examples/http/server.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      gRPC://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+const http = require('http');
+const tracing = require('@opencensus/nodejs');
+const { plugin } = require('@opencensus/instrumentation-http');
+const { ZipkinTraceExporter } = require('@opencensus/exporter-zipkin');
+const { TraceContextFormat } = require('@opencensus/propagation-tracecontext');
+
+const tracer = setupTracerAndExporters();
+
+/** Starts a HTTP server that receives requests on sample server port. */
+function start (port) {
```

Comment:
Maybe call this `startServer` so it's clearer at first glance? Nit: also extra space before parens

Reply:
Sure, done.